.github/workflows/main.yaml

```diff
@@ -297,3 +297,39 @@ jobs:
           fetch-depth: 0
 
       - run: make image
+
+
+  scan_vulnerabilities:
```

Comment:
It's better to add  a new yaml for this? Will be more clear. @fengshunli @vincentgoat

Reply:
Agree. It's better to add a new yaml in the same dir and move this new job `scan_vulnerabilities` into it. @fengshunli